graphql/query_resolver.go

```diff
@@ -406,7 +406,7 @@ func (r *queryResolver) Patch(ctx context.Context, patchID string) (*restModel.A
 }
 
 // GithubProjectConflicts is the resolver for the githubProjectConflicts field.
-func (r *queryResolver) GithubProjectConflicts(ctx context.Context, projectID string) (*model.GithubProjectConflicts, error) {
+func (r *queryResolver) GithubConflicts(ctx context.Context, projectID string) (*model.GithubProjectConflicts, error) {
```

Comment:
This is used in a generated file and I think since it's updated here but not the generated file ("graphql/generated.go") it's causing the tests to fail from not compiling `[2024/05/01 10:02:04.493] graphql/query_resolver.go:1011:51: cannot use &queryResolver{…} (value of type *queryResolver) as QueryResolver value in return statement: *queryResolver does not implement QueryResolver (missing method GithubProjectConflicts)`

Reply:
Also haven't touched graphql that much but should the model name change too?

Reply:
Oh good catch, don't think I meant to make this change; removing it